data/features/flexbox-gap.js

```diff
@@ -8,11 +8,21 @@
  * @type {import('../features').Feature}
  */
 export default (rule) => {
-  if (/display:\s(inline-)?flex/.test(rule.toString())
-    && rule.toString().includes('gap:')
-  ) {
-    return true;
+  if (!('some' in rule)) {
+    return false;
   }
 
-  return false;
+  let isFlexbox = false;
+  let isGap = false;
+
+  return rule.some((decl) => {
+    if (decl.type !== 'decl') {
+      return false;
+    }
+
+    isFlexbox ||= decl.prop === 'display' && (decl.value === 'flex' || decl.value === 'inline-flex');
+    isGap ||= decl.prop === 'gap' || decl.prop.endsWith('-gap');
```

Comment:
Just use `column-gap` and `row-gap` directly since it's faster and eliminates any possible future problems. If there's a new property later, it'll need it's own feature rule anyway.